AspNetCoreApplication/DTO/DTObook/BookItem.cs

```diff
@@ -21,14 +20,10 @@ public class BookItem
 
         public int AuthorId { get; set; }
 
-        public Author Author { get; set; }
-
-        public int UserId { get; set; }
-
-        public User User { get; set; }
-
         public string Cover { get; set; }
 
-        public List<BookCategory> Categories { get; set; }
+        public List<Category> Categories { get; set; }
```

Comment:
Ở đây phải là CategoryItem để không được phép có `books` 
![image](https://user-images.githubusercontent.com/10450325/128292262-ef15d0b1-21ca-4337-888a-ce5e2069818d.png)

Reply:
```suggestion
        public List<CategoryItem> Categories { get; set; }
```

Và CategoryItem thì không có Books

Reply:
done